package.json

```diff
@@ -100,8 +101,6 @@
     "msw": "^0.35.0",
     "np": "^7.5.0",
     "omit-deep-lodash": "^1.1.5",
-    "react": "^17.0.2",
-    "react-dom": "^17.0.2",
```

Comment:
These 2 need to stay in dev dependencies, otherwise they won't be installed and tests would fail.